docs/using.rst

```diff
@@ -8,14 +8,16 @@ User Guide
 Getting Certbot
 ===============
 
-To get specific instructions for installing Certbot on your OS, we recommend
-visiting certbot.eff.org_. If you're offline, you can find some general
+To get specific instructions for installing Certbot on your OS,
+visit certbot.eff.org_. This is the easiest way to install Certbot. 
```

Comment:
I wonder if this should be something like "This is the easiest way to learn how to get Certbot up and running on your system." I'm suggesting this because visiting the website only gives you instructions on how to do it and doesn't actually install the software. If you think clarifying this sentence isn't necessary, feel free to ignore this comment.